content/docs/1.7.0/important-notes/_index.md

```diff
@@ -35,6 +36,18 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
   - [Snapshot Creation Time As Shown in the UI Occasionally Changes](#snapshot-creation-time-as-shown-in-the-ui-occasionally-changes)
   - [Unable To Revert a Volume to a Snapshot Created Before Longhorn v1.7.0](#unable-to-revert-a-volume-to-a-snapshot-created-before-longhorn-v170)
 
+## Warning
+
+> **PSA**
```

Comment:
Remove this. PSA is just used for creating messages to the community Slack channels.